manila/share/drivers/netapp/dataontap/client/client_cmode.py

```diff
@@ -4086,18 +4125,61 @@ def prune_deleted_volumes(self):
                     .get_child_by_name('volume-id-attributes')
                     .get_child_content('owning-vserver-name'))
 
+            LOG.debug('Found volume %(vv)s in state  %(vs)s', {
+                      'vv': volume_name, 'vs': volume_state})
             if volume_state == 'offline':
+                clones = self.get_clones_of_parent_volume(vserver, volume_name)
+                if clones:
+                    # Found parent volume which has multiple clone childs, so
+                    # make volume online and start clone splits.
```

Comment:
Can you make it clearer in the comment, that the clone split will intentionally be done in the next call,
because we need to give the 'online-operation' some time to finish?